packages/cli/README.md

```diff
@@ -0,0 +1,72 @@
+# Prosopo Provider Command Line Interface
```

Comment:
Unless this is just for internal use then this should have a little more context imo

What does fund do, why should I do it, how much should I do it, how do i get dataset.json, (etc that kind of context)